### ISSUE TITLE
Crash after calling game from Recent apps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#274]
```
Steps to reproduce:
1. Enter the game and go to the skills screen.
2. Press Home button.
3. Launch game from recent apps.

Actual results:
3. Crash.

Logcat:
11-13 12:55:25.924: E/AndroidRuntime(14695): FATAL EXCEPTION: main
11-13 12:55:25.924: E/AndroidRuntime(14695): java.lang.RuntimeException: Unable 
to start activity 
ComponentInfo{com.gpl.rpg.AndorsTrail/com.gpl.rpg.AndorsTrail.activity.HeroinfoA
ctivity}: java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.gpl.rpg.AndorsTrail/com.gpl.rpg.AndorsTrail.activity.HeroinfoA
ctivity_Stats}: java.lang.NullPointerException
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1872)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:1893)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.access$1500(ActivityThread.java:135)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread$H.handleMessage(ActivityThread.java:1054)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.os.Handler.dispatchMessage(Handler.java:99)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.os.Looper.loop(Looper.java:150)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.main(ActivityThread.java:4385)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
java.lang.reflect.Method.invokeNative(Native Method)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
java.lang.reflect.Method.invoke(Method.java:507)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:849)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
com.android.internal.os.ZygoteInit.main(ZygoteInit.java:607)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
dalvik.system.NativeStart.main(Native Method)
11-13 12:55:25.924: E/AndroidRuntime(14695): Caused by: 
java.lang.RuntimeException: Unable to start activity 
ComponentInfo{com.gpl.rpg.AndorsTrail/com.gpl.rpg.AndorsTrail.activity.HeroinfoA
ctivity_Stats}: java.lang.NullPointerException
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1872)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.startActivityNow(ActivityThread.java:1692)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.LocalActivityManager.moveToState(LocalActivityManager.java:127)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.LocalActivityManager.startActivity(LocalActivityManager.java:339)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.widget.TabHost$IntentContentStrategy.getContentView(TabHost.java:656)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.widget.TabHost.setCurrentTab(TabHost.java:326)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.widget.TabHost.addTab(TabHost.java:216)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
com.gpl.rpg.AndorsTrail.activity.HeroinfoActivity.onCreate(HeroinfoActivity.java
:33)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1072)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1836)
11-13 12:55:25.924: E/AndroidRuntime(14695):    ... 11 more
11-13 12:55:25.924: E/AndroidRuntime(14695): Caused by: 
java.lang.NullPointerException
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
com.gpl.rpg.AndorsTrail.activity.HeroinfoActivity_Stats.onCreate(HeroinfoActivit
y_Stats.java:51)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1072)
11-13 12:55:25.924: E/AndroidRuntime(14695):    at 
android.app.ActivityThread.performLaunchActivity(ActivityThread.java:1836)
11-13 12:55:25.924: E/AndroidRuntime(14695):    ... 20 more

```

Original issue reported on code.google.com by `prehisto...@gmail.com` on 13 Nov 2011 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
Could not reproduce the reported behavior on my DROIDX 2.3.3
```

Original comment by `jgemmajr@gmail.com` on 16 Nov 2011 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report! This should now be fixed in r218.
```

Original comment by `oskar.wi...@gmail.com` on 12 Feb 2012 at 8:06
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
